docs/book/tutorials/kubernetes-on-vsphere-with-kubeadm.md

```diff
@@ -156,262 +208,994 @@ bootstrapTokens:
+daemonset.extensions/vsphere-cloud-controller-manager created
+service/vsphere-cloud-controller-manager created
+```
+### Verify that the CPI has been successfully deployed
+Verify vsphere-cloud-controller-manager is running and all other system pods are up and running (note that the coredns pods were not running previously - they should be running now as the taints have been removed by installing the CPI):
+```
+# kubectl get pods --namespace=kube-system
+NAME                                     READY   STATUS    RESTARTS   AGE
+coredns-fb8b8dccf-bq7qq                  1/1     Running   0          71m
+coredns-fb8b8dccf-r47q2                  1/1     Running   0          71m
+etcd-k8s-master                          1/1     Running   0          69m
+kube-apiserver-k8s-master                1/1     Running   0          70m
+kube-controller-manager-k8s-master       1/1     Running   0          69m
+kube-flannel-ds-amd64-7kmk9              1/1     Running   0          38m
+kube-flannel-ds-amd64-dtvbg              1/1     Running   0          63m
+kube-flannel-ds-amd64-hq57c              1/1     Running   0          30m
+kube-flannel-ds-amd64-j7g4s              1/1     Running   0          22m
+kube-flannel-ds-amd64-q4zsn              1/1     Running   0          21m
+kube-proxy-6jcng                         1/1     Running   0          30m
+kube-proxy-bh8kh                         1/1     Running   0          21m
+kube-proxy-rb9xp                         1/1     Running   0          22m
+kube-proxy-srhpj                         1/1     Running   0          71m
+kube-proxy-vh4lg                         1/1     Running   0          38m
+kube-scheduler-k8s-master                1/1     Running   0          70m
+vsphere-cloud-controller-manager-549hb   1/1     Running   0          25s
+```
+Check that all nodes are untainted
+Verify node.cloudprovider.kubernetes.io/uninitialized taint is removed from all nodes.
+```
+# kubectl describe nodes | egrep "Taints:|Name:"
+Name:               k8s-master
+Taints:             node-role.kubernetes.io/master:NoSchedule
+Name:               k8s-node1
+Taints:             <none>
+Name:               k8s-node2
+Taints:             <none>
+Name:               k8s-node3
+Taints:             <none>
+Name:               k8s-node4
+Taints:             <none>
+```
+
+Note: If you happen to make an error with the vsphere.conf, simply delete the CPI components and the configMap, make any necessary edits to the configMap vSphere.conf file, and reapply the steps above.
 
-### Verify setup
+## Install vSphere Container Storage Interface Driver
+Now that the CPI is installed, we can focus on the CSI. Perform the following steps on the Master Node(s) only.
+Taint the master nodes to prevent scheduling
+The node-role.kubernetes.io/master=:NoSchedule taint is required to be present on the master nodes to prevent scheduling of the node plugin pods for vsphere-csi-node daemonset on the master nodes.
```

Comment:
Add backticks around this too

Reply:
I haven't experienced this in my labs/testing? IMO - we should leave this out incase people allow normal pods to run on their masters?

Reply:
Hmm - I did this during the install - let's validate it once more before making a decision.